lib/elements/custom-style.js

```diff
@@ -75,7 +75,7 @@ export class CustomStyle extends HTMLElement {
    * @export
    * @return {HTMLStyleElement} This element's light-DOM `<style>`
    */
-  getStyle() {
+  'getStyle'() {
```

Comment:
Was this change intentional? It causes an error `@export is not supported on this expression.`

Reply:
Good point, export should be sufficient.  Will revert back to `getStyle()`